tests/php/ORM/DBMoneyTest.php

```diff
@@ -292,7 +292,7 @@ public function testWriteToDataObject()
             )->value()
         );
         $this->assertEquals(
-            '1.23',
+            '1.2300',
```

Comment:
As above, this only regressed once statement caching was removed. However, I’d argue that the new test is correct and the old one isn’t: `MyMoneyAmount` is a `DECIMAL(19, 4)`.